pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -28,9 +29,18 @@ func availabilityZones(ctx context.Context, session *session.Session, region str
 		return nil, errors.Wrap(err, "fetching availability zones")
 	}
 
+	// This regex finds local zones and allows us to filter them out.
+	// This is a temporary fix until we update our aws-sdk-go package to a
+	// version that adds the ZoneType field to the AvailabilityZone struct
+	regex, err := regexp.Compile("^\\w+-\\w+-\\w+-\\w+-\\w+$")
```

Comment:
Will this test hold up for other AWS partitions? For example, in the us gov partition, the regions are `us-gov-east-1` and `us-gov-west-1`. Presumably, if c2s did support local zones, then the zone name would have 5 hyphens instead of 4. Should we instead check the number of parts after the region name in the zone name?

Reply:
The local zones are currently only in us-east-1 and us-west-2. IMHO this is sufficient to hold us over until we are able to update the packages and provide a more permanent solution.

Reply:
Yeah this makes sense to me